db/db.connect.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const username = process.env['DB_USERNAME'];
+const password = encodeURIComponent(process.env['DB_PASSWORD']);
+
+
+async function initializeDBConnection() {
+    const uri = `mongodb+srv://${username}:${password}@neog-cluster.7up1q.mongodb.net/videolibrary?retryWrites=true&w=majority`;
```

Comment:
you can just save the url itself as env variable so that you don't have to use two variables ie username and password